modules/p2/ding_interaction/plugins/content_types/interaction_pane.inc

```diff
@@ -32,12 +32,6 @@ function ding_interaction_admin_info($subtype, $conf, $context) {
  * Implements hook_content_type_render().
  */
 function ding_interaction_content_type_render($subtype, $conf, $panel_args, $context) {
-  // Only show on front if serendipity is enabled for the frontpage.
-  // See ding_frontpage config at /admin/config/ding/frontend.
-  if (drupal_is_front_page() && !variable_get('ding_frontpage_serendipity') && variable_get('ding_frontpage_promoter') != 'serendipity') {
-    return;
-  }
-
```

Comment:
Why has this been removed. It does not seem to be related to the change at hand.